src/tpm2.c

```diff
@@ -34,7 +34,13 @@
 /* --- Local Variables -- */
 /******************************************************************************/
 
-static THREAD_LS_T TPM2_CTX* gActiveTPM;
+#if defined(WOLFTPM_SWTPM_UARTNS550)
```

Comment:
Would prefer a different build option to make this a non-static non-thread local. Maybe `WOLFTPM2_PUBLIC_CTX`?